classes/simplemap.php

```diff
@@ -588,7 +588,7 @@ function add_taxonomy_fields( $atts, $taxonomy ) {
 			} elseif ( 'select' == $taxonomy_field_type ) {
 				// Print selectbox if that's what we're doing.
 				$tax_select = "<select id='location_search_" . esc_attr( $tax_field_name ) . "_select' name='location_search_" . esc_attr( $tax_field_name ) . "_select' >";
-				$tax_select .= "<option value=''>" . apply_filters( 'sm-search-tax-select-default', __( 'Select a value', 'simplemap' ), $taxonomy ) . '</option>';
+				$tax_select .= "<option value=''>" . apply_filters( 'sm-search-tax-select-default', __( 'Select a value', 'SimpleMap' ), $taxonomy ) . '</option>';
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 163 characters

<!-- Comment by [SideCI](https://sideci.com) -->